src/xcode/ENA/ENA/Source/Scenes/Events/Store/EventStore.swift

```diff
@@ -12,7 +12,8 @@ import FMDB
 // swiftlint:disable:next type_body_length
 class EventStore: SecureSQLStore, EventStoringProviding {
 
-	static let dataRetentionPeriodInDays = 15
+	static let tenMinutesIntervalLength: TimeInterval = 600
+	static let dataRetentionPeriodInDays: TimeInterval = 15
```

Comment:
A bit unusual data type, since a `TimeInterval` is in seconds. Maybe you could use Double instead, but not blocking for this.